net-misc/seafile-client/seafile-client-9.0.11.ebuild

```
@@ -0,0 +1,54 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Upstream is moving tags repeatedly, then we use commit hash.
+RELEASE_COMMIT="4088a6c394e7f2f785d2f5e175a4e916259bdb09"
+
+inherit xdg cmake
+
+DESCRIPTION="Seafile desktop client"
+HOMEPAGE="https://www.seafile.com/ https://github.com/haiwen/seafile-client/"
+SRC_URI="https://github.com/haiwen/${PN}/archive/${RELEASE_COMMIT}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${RELEASE_COMMIT}"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="shibboleth test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-db/sqlite:3
+	dev-libs/glib:2
+	dev-libs/jansson:=
+	dev-libs/openssl:=
+	dev-qt/qtcore:5
+	dev-qt/qtdbus:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtwidgets:5
```

```suggestion
	dev-qt/qtbase:6[dbus,gui,network,ssl,widgets]
	dev-qt/qtwebengine:6[widgets]
	dev-qt/qt5compat:6
```

Why not switch to Qt6 ? Build is ok with -DQT_VERSION_MAJOR=6. Only test failed (still Qt5).

---

> it seems test failed for current stable version too.

Maybe restrict it then.

---

There is no such thing as a half-switch, anyway. The first version of your PR, switching only dev-qt/qtwebengine to SLOT 6, would not have worked at all.

How did you test that?